aten/src/ATen/native/cudnn/BatchNorm.cpp

```diff
@@ -35,17 +35,24 @@ std::tuple<Tensor, Tensor, Tensor> cudnn_batch_norm_backward(
   AT_ERROR("cudnn_batch_norm_backward: ATen not compiled with cuDNN support");
 }
 
+size_t _get_cudnn_batch_norm_reserve_space_size(
+    const Tensor& input_t,
+    bool training) {
+  AT_ERROR(
+      "_get_cudnn_batch_norm_reserve_space_size: ATen not compiled with cuDNN support");
```

Comment:
Is this actually needed? Now that you are using macros to not call this fn in the first place when cudnn is disabled.

Reply:
Yes, I found out that this file is actually always included when building with cuda, regardless of whether cudnn is available. Without this definition the tests would fail when building on cuda machines (without cudnn), complaining that the symbol is not defined.